client/onboarding/step.tsx

```diff
@@ -32,7 +31,6 @@ const Step: React.FC< Props > = ( { name, children } ) => {
 
 	return (
 		<>
-			<div className="stepper__progress" style={ { width } } />
```

Comment:
oh no :( I liked progress

Reply:
It wasn't in the design, which is why I removed it - I guess it probably isn't as necessary now that we have only 2 steps.

cc. @elizaan36 just to double check this!

Reply:
So... The reason I think the progress bar doesn't work as well for these two screens is it's somewhat misleading. I think the progress bar would only really work if it was continuous through a contained flow, from 1. WordPress.com connection, 2. MOX, 3. Stripe verification. I'm open to other ideas though, if you think it's essential for this section of the onboarding!

Reply:
Thanks for sharing the thinking behind it. I agree it can be misleading because of the extra steps like Jetpack Connection and Stripe KYC. Will have a think about if there's another way to show progress, but given the MOX is now only 2 steps it doesn't seem necessary imo :)